app/routes/index/chapter/results.js

```diff
@@ -12,7 +12,7 @@ export default Ember.Route.extend({
   model() {
     return Ember.RSVP.hash({
       member: this.modelFor('index').member,
-      chapter: this.modelFor('index').chapter,
+      chapter: this.modelFor('index/chapter').chapter,
```

Comment:
This was the actual error. You could have also used the route's chapter ID param to fetch the chapter here if you only wanted to fix the bug.